rncryptor.cabal

```diff
@@ -31,8 +31,8 @@ library
     , random >= 1.0.0.1
     , QuickCheck >= 2.6 && < 2.9
     , io-streams >= 1.2.0.0
-    , cipher-aes >= 0.2.9
-    , pbkdf >= 1.1.1.1
+    , cryptonite
```

Comment:
adding cryptonite >= 0.15 gave me a stack error:

--  Failure when adding dependencies:    
      cryptonite: needed (>=0.15), 0.6 found (latest applicable is 0.19)
    needed for package rncryptor-0.0.2.3

Adding cryptonite >= 0.6 works.  Is that ok?